tests/functional/object/mcg/test_noobaa_bucket_error_state_mode.py

```
@@ -0,0 +1,481 @@
+        backingstore = bucket.bucketclass.backingstores[0]
+
+        logger.info(
+            f"Created data bucket {bucket.name} with "
+            f"17Gi PV pool backing store {backingstore.name}"
+        )
+
+        observed_modes = []
+        file_num = 0
+        final_mode = None
+
+        testfile = "/tmp/testfile"
```

Ruff (0.15.17)

[error] Probable insecure usage of temporary file or directory: "/tmp/testfile"

(S108)